docs/admin/network-plugins.md

```diff
@@ -43,3 +43,10 @@ The plugin requires a few things:
 * Kubelet must be run with the `--network-plugin=kubenet` argument to enable the plugin
 * Kubelet must also be run with the `--reconcile-cidr` argument to ensure the IP subnet assigned to the node by configuration or the controller-manager is propagated to the plugin
 * The node must be assigned an IP subnet through either the `--pod-cidr` kubelet command-line option or the `--allocate-node-cidrs=true --cluster-cidr=<cidr>` controller-manager command-line options.
+
+## Usage Summary
+
+* `--network-plugin=exec`: Must specify the `--network-plugin-dir` flag to the location of executables.
+* `--network-plugin=cni`: Specify the `--network-plugin-dir` flag to the location of CNI config file. Default to `/etc/cni/net.d`. Meanwhile, CNI binaries must be present in `/opt/cni/bin`.
```

Comment:
Something like: specifies that we use the cni plugin with cin plugin configuration located in `network-plugin-dir` and actual plugin binaries located in ..., config location defaults to ..., binary location defaults to